velero-plugins/migscc/restore.go

```diff
@@ -0,0 +1,65 @@
+package migscc
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/heptio/velero/pkg/plugin/velero"
+	apisecurity "github.com/openshift/api/security/v1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// RestorePlugin is a restore item action plugin for Velero
+type RestorePlugin struct {
+	Log logrus.FieldLogger
+}
+
+// AppliesTo returns a velero.ResourceSelector that applies to PVCs
+func (p *RestorePlugin) AppliesTo() (velero.ResourceSelector, error) {
+	return velero.ResourceSelector{
+		IncludedResources: []string{"securitycontextconstraints"},
```

Comment:
I believe the `security.openshift.io` should be included as well.

Reply:
What is the difference?

Reply:
security.openshift.io is the group part of the apViersion. All of our other plugins just specify the "kind" field (in lowercase like this), so this is consistent with what we're doing elsewhere. I'm not actually sure whether adding the apiVersion bits would be correct, or what change in behavior it would cause, but I think the way we're doing it here is right.